docs/sources/flow/reference/components/otelcol.connector.servicegraph.md

```diff
@@ -0,0 +1,218 @@
+---
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.connector.servicegraph/
+labels:
+  stage: experimental
+title: otelcol.connector.servicegraph
+---
+
+# otelcol.connector.servicegraph
+
+{{< docs/shared lookup="flow/stability/experimental.md" source="agent" >}}
+
+`otelcol.connector.servicegraph` accepts span data from other `otelcol` components and 
+outputs metrics representing the relationship between various services in a system.
+A metric represents an edge in the service graph.
+Those metrics can then be used by a data visualization application (e.g. 
+[Grafana](https://grafana.com/docs/grafana/latest/explore/trace-integration/#service-graph))
+to draw the service graph.
+
+> **NOTE**: `otelcol.connector.servicegraph` is a wrapper over the upstream
+> OpenTelemetry Collector `servicegraph` connector. Bug reports or feature requests
+> will be redirected to the upstream repository, if necessary.
+
+Multiple `otelcol.connector.servicegraph` components can be specified by giving them
+different labels.
+
+This component is based on [Grafana Tempo's service graph processor](https://github.com/grafana/tempo/tree/main/modules/generator/processor/servicegraphs).
+
+Service graphs are useful for a number of use-cases:
+
+* Infer the topology of a distributed system. As distributed systems grow, they become more complex. 
+  Service graphs can help you understand the structure of the system.
+* Provide a high level overview of the health of your system.
+  Service graphs show error rates, latencies, and other relevant data.
+* Provide a historic view of a system’s topology.
+  Distributed systems change very frequently,
+  and service graphs offer a way of seeing how these systems have evolved over time.
+
+Since `otelcol.connector.servicegraph` has to process both sides of an edge,
+it needs to process all spans of a trace to function properly.
+If spans of a trace are spread out over multiple Agent instances, spans cannot be paired reliably.
+A solution to this problem is using [otelcol.exporter.loadbalancing](./otelcol.exporter.loadbalancing.md)
```

Comment:
Thanks for checking it! I'll also change it in `otelcol.connector.spanlogs.md`, since I'd put a similar link there and now that I checked it on our website, it doesn't work. Interestingly, it does work in the VS code markdown preview windows 🤷‍♂️

Reply:
I ran `make docs` just now, and I can confirm that both links work now. Thanks again!